build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-turing/build.sh

```diff
@@ -70,13 +70,14 @@ echo "Building with Ninja"
 cd "${CMAKE_BUILD_DIR?}"
 ninja
 
-# Limit parallelism to 8 to avoid exhausting GPU memory
+# Limit parallelism dramatically to avoid exhausting GPU memory
 # TODO(#5162): Handle this more robustly
-DEFAULT_PARALLELISM="$(nproc)"
-if [[ "${DEFAULT_PARALLELISM?}" -gt "8" ]]; then
-   DEFAULT_PARALLELISM=8
-fi
-export CTEST_PARALLEL_LEVEL=${CTEST_PARALLEL_LEVEL:-${DEFAULT_PARALLELISM?}}
+export CTEST_PARALLEL_LEVEL=${CTEST_PARALLEL_LEVEL:-1}
 
+# Run only the vulkan tests, since these are the only ones for which running on
+# a machine with a GPU should be different.
 echo "Testing with CTest"
-ctest --output-on-failure -L 'integrations/tensorflow' --label-exclude "^nokokoro$"
+ctest --output-on-failure \
+   --tests-regex "^integrations/tensorflow" \
+   --label-regex "^driver=vulkan$" \
```

Comment:
and CUDA?

Reply:
We don't have any cuda integration tests (yet). I can add it as a precautionary measure.